R/dettl_config.R

```diff
@@ -58,6 +58,12 @@ db_config_read_yaml <- function(filename, path) {
     info$db[[db_cfg]]$driver <- check_symbol_from_str(
       info$db[[db_cfg]]$driver, sprintf("%s:%s:driver", filename, db_cfg)
     )
+    if (!is.null(info$db[[db_cfg]]$log_table)) {
```

Comment:
I think it will be necessary to have the table configurable, as you have done here.

However, it's probably not something that can be *changed* once configured, or if so requires manually moving the previous table around.  Can we make sure that is appropriately documented and consider removing the possibility of a default (a default vault means that a move is more likely, a required value reduces the chance that people will want to move the table later)

Reply:
Removed the default and I've added example and note to the vignette. Though I will flesh this out when I come to impl the work that actually does the logging.